chord_metadata_service/chord/workflows/wdls/experiments_json.wdl

```diff
@@ -25,3 +40,30 @@ task copy_task {
         File json_document = "ingest.json"
     }
 }
+
+task ingest_task {
+    input {
+        File json_document
+        String service_url
+        String dataset_id
+        String token
+    }
+    command <<<
+        RESPONSE=$(curl -X POST -k -s -w "%{http_code}" \
+            -H "Content-Type: application/json" \
+            -H "Authorization: ~{token}" \
+            --data "@~{json_document}" \
+            "~{service_url}/ingest/~{dataset_id}/experiments_json")
+        if [ "${RESPONSE}" != "204" ]
+        then
+            echo "Error: Metadata service replied with HTTP code ${RESPONSE}" 1>&2  # to stderr
+            exit 1
```

Comment:
nice error handling! but maybe worth logging the complete response value rather than just the code if the ingestion fails (not sure if its possible to coerce CURL to do this nicely in just bash...)